docs/reference_guide.md

```diff
@@ -1782,17 +1782,20 @@ int main()
 bpftrace will output "hit" and exit when the memory pointed to by `arg1` of `increment` is
 written.
 
-## 17. `kfunc`/`kretfunc`: Kernel Functions Tracing
+## 17. `kfunc`/`kretfunc` (alias: `fentry`/`fexit`): Kernel Functions Tracing
 
 Syntax:
 
 ```
 kfunc[:module]:function
 kretfunc[:module]:function
+fentry[:module]:function
+fexit[:module]:function
```

Comment:
Super nit: interleave?

Reply:
You mean like `kfunc/fentry[:module]:function` ?

Reply:
Oh sorry, I meant like:

```
kfunc[:module]:function
fentry[:module]:function
kretfunc[:module]:function
fexit[:module]:function

```